tools/marketplace/helper/bundleHelper.go

```diff
@@ -158,7 +158,8 @@ func (bh *BundleHelper) addOldCRDs(outDir string) error {
 
 	currentVersion := v1.KubeVirtGroupVersionKind.Version
 	for _, crd := range bh.CRDs {
-		if crd.Spec.Version == currentVersion {
+		// look for the latest CRD version which is the 1st one in the list
```

Comment:
Is this guaranteed?

Reply:
Actually, you're right, it's not https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#version-priority.

I don't know why but I assumed it was like this, I'll think of something else.

Reply:
Updated.